draft-ietf-httpbis-encryption-encoding.md

```diff
@@ -359,13 +359,13 @@ header field.
 
 aesgcm:
 
-: The "aesgcm" parameter contains the URL-safe base64 [RFC4648] octets of the
-input keying material.
+: The "aesgcm" parameter contains the URL and filename safe base64 [RFC4648]
+  octets of the input keying material with padding removed.
```

Comment:
Do you want to be strict in regards to omitting padding? Right now our implementation is lenient and just ignores it.